### ISSUE TITLE
Crashes when colon in search term

### ISSUE THREAD

**Issue by 2016rshah** [2016rshah/bookshelf#26]
If you try to search for a book with a colon in the title it will crash. 

Reproduce by trying to search for 

`Chaos Monkeys: Obscene Fortune and Random Failure in Silicon Valley` 

with and without the colon

**Comment by MasterOfTheTiger** [comment]
You can make the engine remove the colon.

**Comment by 2016rshah** [comment]
Yeah, I'm treating this issue as a todo list item but I'm not sure if I'm gonna get around to fixing it any time soon.

If you want to send in a pull request I'd really appreciate it!

**Comment by MasterOfTheTiger** [comment]
Okay. I will have to learn how to do it though. I want to fix it.

**Comment by MasterOfTheTiger** [comment]
Where is the code for it?

**Comment by 2016rshah** [comment]
It's a standard Ruby on Rails project, so that would be a good place to start learning! 

The main code is in `app/`, you're probably looking for the code in `app/controllers/` like [the books controller](https://github.com/2016rshah/bookshelf/blob/master/app/controllers/books_controller.rb) and probably the search function.

**Comment by MasterOfTheTiger** [comment]
Okay! I’ve been wanting to learn Ruby on Rails.
